File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/plugins/angularjs/service/sellerService.js
app.service('sellerService', function ($http) {

    this.findByPage = function (page, size, searchEntity) {
        return $http.post('../seller/findByPage.do?page=' + page + '&size=' + size, searchEntity);
    };

    this.findById = function (id) {
        return $http.get('../seller/findById.do?id=' + id);
    };

    this.updateStatus = function (ids, status) {
        return $http.get('../seller/updateStatus.do?ids=' + ids + '&status=' + status);
    };
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/SellerService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSeller;
import entity.PageResult;

public interface SellerService {

    PageResult<TbSeller> findByPage(TbSeller seller, int pageNum, int pageSize) throws Exception;

    TbSeller findById(String id) throws Exception;

    void add(TbSeller seller) throws Exception;

    void updateStatus(String id, String status) throws Exception;
}